csufmse/student_information_system
# auto create semesterstudent, fixed semester test data, and added some…

… stuff to /schooladmin/user to help testing

Fixes #171.

Testing:
- Verify the semesters (in /schooladmin/semesters) have date_registraion_opens < other dates
- Verify that in /student/registration you can only select the 3 semesters that are "open" (date_reg < today < date_ended)
- Verify that after you've registered student, the courses show up in /schooladmin/user on the user page in course history.
